src/documentation/templates/plan-calculator.html

```diff
@@ -232,14 +242,29 @@ <h4>Contactos</h4>
                               type="range"
                               min="0"
                               value="0"
-                              max="5"
+                              max="12"
                               step="1"
                             />
                             <div class="progress-anchor"></div>
-                            <div class="dp-indicator">
-                              <span><strong>1500</strong></span
-                              ><span><strong>100k</strong></span>
-                            </div>
+                            <ul class="datalist">
+                              <li>
+                                <span><strong>100K</strong></span>
+                              </li>
+                              <li></li>
+                              <li></li>
+                              <li></li>
+                              <li></li>
+                              <li></li>
+                              <li></li>
+                              <li></li>
+                              <li></li>
+                              <li></li>
+                              <li></li>
+                              <li></li>
```

Comment:
Why did you keep these empty?

Reply:
in case there are more than 8 plans to be shown in the slider

Reply:
<img width="417" alt="Captura de Pantalla 2022-10-11 a la(s) 1 50 03 p  m" src="https://user-images.githubusercontent.com/84402180/195174841-693aab13-4c6c-4aed-b67e-9b9e0e551992.png">

Reply:
Thanks!

A related question, should not the number of plans be dynamic or data based?

Reply:
Yes, it's dynamic

Reply:
If it is dynamic, why did you add 13 `li` manually?

Reply:
Sorry, they are harcoded in style guide ... 
But, they will be dynamic in the webapp.